### PR TITLE
[None][perf] Support Gemma RMSNorm + interleaved mRoPE in fused_qk_no…

### PR DESCRIPTION
**Pull request by nv-guomingz**
…rm_rope

Qwen3.5 / Qwen3-Next gated full-attention previously ran QK-norm and RoPE as separate kernels (split + 2x RMSNorm + MRotaryEmbedding RoPE) because the fused_qk_norm_rope kernel only supported plain RMSNorm and single-position RoPE. Two blockers are lifted:

- Gemma-style RMSNorm: scale by (1 + weight) via a use_gemma flag.
- Interleaved mRoPE: select the RoPE position component per rotary half-dim (mirrors MRotaryEmbedding.apply_interleaved_rope) via use_mrope + mrope_section, with position_ids passed as [3, num_tokens].

A Gemma+mRoPE unit test is added but skipped pending a fix to its standalone MRotaryEmbedding reference harness.

## Release Notes

* **New Features**
  * Added support for interleaved mRoPE (multi-head rotary position embedding) enabling flexible per-dimension position encoding
  * Integrated Gemma-style RMSNorm scaling into fused QK normalization for optimized model inference

* **Tests**
  * Added test coverage validating Gemma-style normalization combined with interleaved mRoPE

## PR Checklist

Please review the following before submitting your PR:
- PR description clearly explains what and why. If using CodeRabbit's summary, please make sure it makes sense.
- PR Follows [TRT-LLM CODING GUIDELINES](https://github.com/NVIDIA/TensorRT-LLM/blob/main/CODING_GUIDELINES.md) to the best of your knowledge.
- Test cases are provided for new code paths (see [test instructions](https://github.com/NVIDIA/TensorRT-LLM/tree/main/tests#1-how-does-the-ci-work))
- If PR introduces API changes, an appropriate PR label is added - either `api-compatible` or `api-breaking`. For `api-breaking`, include `BREAKING` in the PR title.
- Any new dependencies have been scanned for license and vulnerabilities
- [CODEOWNERS](https://github.com/NVIDIA/TensorRT-LLM/blob/main/.github/CODEOWNERS) updated if ownership changes
- Documentation updated as needed
- Update [tava architecture diagram](https://github.com/NVIDIA/TensorRT-LLM/blob/main/.github/tava_architecture_diagram.md) if there is a significant design change in PR.
- The reviewers assigned automatically/manually are appropriate for the PR.

- [x] Please check this after reviewing the above items as appropriate for this PR.

## GitHub Bot Help

To see a list of available CI bot commands, please comment `/bot help`.

### CODE REVIEW

**Review by byshiue — APPROVED** [review]

File: tests/unittest/_torch/thop/parallel_hw_agnostic/test_fused_qk_norm_rope.py [hunk 1]
```diff
@@ -204,3 +208,135 @@ def test_fused_qk_norm_rope(
...
+        mrope_section=mrope_section,
+        is_neox=is_neox,
+        mrope_interleaved=True,
+    ).to(qkv.device)
+    [q_rope, k_rope] = rotary_emb(position_ids_3d, [q_n, k_n])
+    return torch.cat([q_rope, k_rope, v], dim=1)
+
+
+@pytest.mark.skip(
+    reason="WIP: standalone MRotaryEmbedding reference shape handling needs "
```
Comment (Wanli-Jiang) [inline tests/unittest/_torch/thop/parallel_hw_agnostic/test_fused_qk_norm_rope.py:261]: better to enable this test to make the new feature is guarded by unittest.

**Review by Wanli-Jiang — APPROVED** [review]

**Review by leslie-fang25 — APPROVED** [review]
